### DISCUSSION TITLE
Sqlean extension

### DISCUSSION THREAD

**Discussion by ilyapl82** [dbeaver/dbeaver#18397] [Q&A]
Hi,

I'm a newbie trying to get sqlean (a statistics package for sqlite ) to work on a dbeaver.

I don't understand what steps I need to take to get it installed on my windows PC and running on my sql code.

I was hoping you could help me with some step by step instructions or a video.

This sqlean extension is on github not marketplace....

https://github.com/nalgeon/sqlean/

Thanks

**Comment by bobromeo** [comment]
I can answer this if it still wanted?

**Reply by ericbf** [reply]
Yes, it is still wanted.

**Comment by mkgrgis** [comment]
@ilyapl82, try to execute standard SQLite command to load extension and work in context with needed functions.

**Comment by bobromeo** [comment]
@ericbf, hi Eric, btw SQLean is not a _statistics_ package :-)

```
select * from artist where name like 'AC%';
select load_extension('D:\Programs\SQLean\text.dll');
select name, text_reverse(name) revname
from artist where name like 'AC%';
```

**Reply by ericbf** [reply]
I understand it's not 🤪, just looking for a way to auto load it for all SQLite connections automatically.
